Octokit/Helpers/AcceptHeaders.cs

```diff
@@ -2,6 +2,14 @@
 {
     public static class AcceptHeaders
     {
+        public const string DefaultJson = "application/vnd.github.quicksilver-preview+json; charset=utf-8, application/vnd.github.v3+json; charset=utf-8";
```

Comment:
This is related to [repository redirects](https://developer.github.com/changes/2015-04-17-preview-repository-redirects/) and then the latest stable version - could we rename it to something like `RedirectsPreviewThenStableVersionJson` ?